src/main/webapp/index.jsp

```diff
@@ -0,0 +1,90 @@
+<% boolean isUserLoggedIn = (boolean) request.getAttribute("isUserLoggedIn"); %>
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>CodeU Starter Project</title>
+    <link rel="stylesheet" href="/css/main.css">
+  </head>
+  <body bgcolor = "#8cdbe2">
```

Comment:
You should definitely coordinate with @gjchong25  and the changes/suggestions in #37 .

Reply:
You are copying and deleting a file she is actively trying to change right now. I'm mainly just saying you need to coordinate with her to determine how you want to proceed so that both sets of changes get reflected in the final outcome. If that means she lands her changes first and then you update your changes then that is fine. If you just promise to integrate her changes into this PR and just close hers then that is fine as well.